test/object-store/CMakeLists.txt

```diff
@@ -100,8 +100,15 @@ if(VSCODE_TEST_RUNNER)
     target_compile_definitions(ObjectStoreTests PRIVATE -DCATCH_CONFIG_CONSOLE_WIDTH=300)
 endif()
 
+# Resources required for running the tests
+file(GLOB REQUIRED_TEST_FILES
```

Comment:
Jeg tænker *.realm vil være tilstrækkeligt.

Reply:
yeah. it was lifted from similar snippet in directory above. Will fix.